frontend/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
```

Comment:
Is this required? Or was it part of the Cypress auto-install?

Reply:
Yeah this was added automatically by Cypress - I would remove, but it will probably get readded by others on `yarn install` :thinking:

Reply:
Makes sense, thanks!